exercises/hamming/hamming_test.exs

```diff
@@ -8,34 +8,39 @@ ExUnit.configure exclude: :pending, trace: true
 defmodule DNATest do
   use ExUnit.Case
 
-  # @tag :pending
+#   @tag :pending
+  test "hamming distance for strands of the same length are ok" do
+    assert {:ok, _} = DNA.hamming_distance('', '')
+  end
```

Comment:
I think the intent was to guide the solution without a major jump, but I'm
also indifferent about it.

On Mon, Jun 20, 2016 at 5:48 PM, Alex <notifications@github.com> wrote:

> In exercises/hamming/hamming_test.exs
> <https://github.com/exercism/xelixir/pull/200#discussion_r67792318>:
>
> > @@ -8,34 +8,39 @@ ExUnit.configure exclude: :pending, trace: true
> >  defmodule DNATest do
> >    use ExUnit.Case
> >
> > -  # @tag :pending
> > +#   @tag :pending
> > +  test "hamming distance for strands of the same length are ok" do
> > +    assert {:ok, _} = DNA.hamming_distance('', '')
> > +  end
>
> this test seems really redundant actually. we'll be checking for {:ok, _}
> in about every test already
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/exercism/xelixir/pull/200/files/51d04cb635c50aca5844604c9af14f475d86b501#r67792318>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAFBHCW8x1ni8ihNIKAfl-ECgE6yIst4ks5qNzTVgaJpZM4I5STM>
> .
>



-- 
*MODE SET**.*
_

jed schneider
developer.

706.207.4458
<http://bittheory.com/>modeset.com <http://www.modeset.com/>